statistics_collector_lib/include/collector/collector.hpp

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef SYSTEM_METRICS_COLLECTOR__COLLECTOR_HPP_
-#define SYSTEM_METRICS_COLLECTOR__COLLECTOR_HPP_
+#ifndef COLLECTOR__COLLECTOR_HPP_
```

Comment:
will need to update this (linter will tell you)

Reply:
Ran `colcon test` and saw no complaints from linter.